### PR TITLE
Use subgraph instead of API

### PR DESCRIPTION
**Pull request by dospore**
Use subgraph query instead of the API temporarily

### CODE REVIEW

File: libs/utils/tracerAPI.ts [hunk 1]
```diff
@@ -43,28 +63,28 @@ export const fetchPoolCommits: (
...
         account?: string;
     },
-) => Promise<APICommitReturn[]> = async (network, { pool, from, to, account }) => {
-    const tracerRoute =
-        `${BASE_TRACER_API}/poolsv2/tradeHistory` +
-        `?network=${network}&from=${from ?? 0}&to=${to ?? Math.floor(Date.now() / 1000)}` +
-        `${!!pool ? `&poolAddress=${pool}` : ''}` +
-        `${!!account ? `&userAddress=${account}` : ''}`;
-    const tracerCommits = await fetch(tracerRoute)
+) => Promise<APICommitReturn[]> = async (_network, { pool, account, from }) => {
```
Comment (ScaredIbis) [inline libs/utils/tracerAPI.ts:66]: is this separate to where we fetch actual trade history? ie its a dupe?

File: libs/utils/tracerAPI.ts [hunk 2]
```diff
@@ -1,28 +1,46 @@
...
 import { CommitEnum } from '@tracer-protocol/pools-js';
 
 export type SourceType = typeof ARBITRUM_RINKEBY | typeof ARBITRUM;
 
-const BASE_TRACER_API = process.env.NEXT_PUBLIC_TRACER_API;
+const query: (props: { pool?: string; account?: string; from?: number }) => string = ({ pool, account, from }) => `{
+    commits (where: { 
+        ${!!pool ? `pool: "${pool.toLowerCase()}",` : ''}
+        ${!!account ? `trader: "${account.toLowerCase()}",` : ''}
+        ${!!from ? `created_gt: ${from},` : ''}
```
Comment (ScaredIbis) [inline libs/utils/tracerAPI.ts:11]: for pending commits you shouldn't need a from, just put `where: { isExecuted: false }`

Comment (dospore) [reply libs/utils/tracerAPI.ts:11]: yep, had that orignally but didnt want to fix the sketchy front running thing, so just dropped it

File: libs/utils/tracerAPI.ts (hunk 1 above)
Comment (dospore) [reply libs/utils/tracerAPI.ts:66]: yiiiis

**Pull request merged**